src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -2651,15 +2522,21 @@ def create_pipeline(
                 )
 
             # Create the pipeline
-            new_pipeline = PipelineSchema.from_create_model(pipeline=pipeline)
+            new_pipeline = PipelineSchema(
+                name=pipeline.name,
+                project_id=pipeline.project,
+                user_id=pipeline.user,
+                docstring=pipeline.docstring,
+                spec=pipeline.spec.json(sort_keys=True),
+            )
```

Comment:
Personally I preferred having this defined in `PipelineSchema.from_create_model(pipeline=pipeline)`, this seems really error-prone now

Reply:
You are right. But there are some objects which require more attributes than what's on the request model. In these case, you would either have to provide additional properties to the `from_create_model` method depending on the schema or pass the `session` to it and make more queries. That's why I stayed away from generalizing it for now.

Reply:
passing additional properties to `from_create_model` seems fine to me